Sources/MapboxSearch/PublicAPI/Search Results/Parking/ParkingRate.swift

```diff
@@ -0,0 +1,16 @@
+@_spi(Experimental)
+public struct ParkingRate: Codable, Hashable {
+    public let maxStayISO8601: String?
+    public let times: [ParkingRateTime]
+    public let prices: [ParkingRatePrice]
```

Comment:
are times and prices connected, e.g. different time period implies a different price? I'm just not sure it would be convenient to have two independent arrays